### ISSUE TITLE
Edit of embed in rich text editor doesn't work

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco-CMS#14256]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

11.0.0

### Bug summary

Previous it was possible to insert an embed in rich text editor, e.g. a YouTube video and selecting this would highlight embed button and allowed to edit the existing embed.
 
![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/ee1dc5e3-afb8-4208-a4be-f7031dba2981)

With Tiny MCE v6 this doesn't seem to be the case anymore.

![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/4946aa48-1f99-424d-b765-05404bb927b3)

### Steps to reproduce

Insert an embed in rich text editor and notice selection of embed doesn't allow to update existing embed, e.g. to update width and height.

**Comment by bjarnef** [comment]
@iOvergaard probably another oversight due to the upgrade of TinyMCE fra v4 to v6.
I noticed this with latest changes in contrib branch (v12 preview).

**Comment by iOvergaard** [comment]
You are most likely correct. I made some changes to that area when upgrading because the old code didn't work out of the box due to changes in the editor API in TinyMCE. There was an issue with the macro picker, which we fixed. We should be able to recognize an embed the same way and highlight the button if one is selected.

I will mark it up for grabs as it's something we could ideally use some help fixing.

Thanks for reporting it, Bjarne!
